sotodlib/preprocess/processes.py

```diff
@@ -1974,28 +1979,45 @@ class GetCommonMode(_Preprocess):
     example config file entry::
 
       - name: "get_common_mode"
+        noise_fit: True
+        f_max: 2.0
+        wrap_name: "common_demodQ"
         calc:
             signal: "signal"
             method: "median"
-            wrap: "signal_commonmode"
+        
         save: True
 
+    If ``noise_fit`` is True, the 1/f noise fit parameters of the common mode
+    is wrapped together.
     .. autofunction:: sotodlib.tod_ops.pca.get_common_mode
     """
     name = 'get_common_mode'
+    def __init__(self, step_cfgs):
+        self.noise_fit = step_cfgs.get('noise_fit', False)
+        self.f_max = step_cfgs.get('f_max', None)
```

Comment:
These are only used in the `calc_and_save` step why do you need to define them globally?